lua/PrePlanManager.lua

```diff
@@ -670,4 +670,10 @@ elseif requiredScript == "lib/managers/preplanningmanager" then
 			QuickMenu:new(managers.localization:text("wolfhud_preplanning_manager_title"), message, { text = managers.localization:text("dialog_ok"), is_cancel_button = true }, true)
 		end
 	end
+elseif requiredScript == "lib/managers/menu/preplanningmapgui" then
+	local set_map_position_to_item_original = PrePlanningMapGui.set_map_position_to_item
+	function PrePlanningMapGui:set_map_position_to_item(...)
+	if item then
+		return set_map_position_to_item_original(self, ...)
+	end
```

Comment:
```lua
function PrePlanningMapGui:set_map_position_to_item(item, ...)
	if item then
		return set_map_position_to_item_original(self, item, ...)
	end
end
```

Reply:
Variable item needs to be defined and passed on.
'end'-statement for if-clause missing.